db/schema.rb

```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_05_10_181721) do
-
+ActiveRecord::Schema.define(version: 2020_06_05_155027) do
```

Comment:
@ThibG According to the docs, the `version`should point to the latest migration, so I simply chose the later date. I hope that's how it's done 🙈

Reply:
That is fine, it doesn't matter much, as those migrations can be applied in any order.